docs/data/pages.ts

```diff
@@ -110,7 +110,7 @@ const pages: MuiPage[] = [
       { pathname: '/x/react-data-grid/performance' },
       {
         pathname: '/x/react-data-grid-group-pivot',
-        title: 'Group & Pivot',
+        title: 'Grouping and pivoting',
```

Comment:
Side note: What do you think about flattening this arbitrary group?

<img width="285" src="https://github.com/user-attachments/assets/6eb305e8-7652-4849-8aff-164774914be7">

Reply:
To give some context, the group was based on AG Grid doc (and existed even before I arrived at MUI and way before any of the feature were implemented).